src/views/v2/month/event-multiday.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * View: Month Event Multiday
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/event-multiday.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+
+$event    = $this->get( 'event' );
+$event_id = $event->ID;
+
+$classes = [ 'tribe-events-calendar-month__event-multiday' ];
+
+if ( $is_featured = tribe( 'tec.featured_events' )->is_featured( $event_id ) ) {
+	$classes[] = 'tribe-events-calendar-month__event-multiday--featured';
+}
+?>
+<div class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>">
```

Comment:
Let's add another inner wrapper to this, for a couple reasons. So our markup will look something like:

```html
<div class="tribe-events-calendar-month__event-multiday">
    <div class="tribe-events-calendar-month__event-multiday-inner">
        <em ...></em>
        <h3></h3>
    </div>
</div>
```

1. When we had our call with Matt about the month view, there was discussion about the 0.1 opacity background and crossing over day blocks and having the grid lines show through the background. We're going to want to add a white background to the outer wrapper and have the blue 0.1 opacity background on the inner wrapper.
2. Matt is going to pick this out, but the border radius implementation isn't quite the same as what the designs are. Using an outer and inner wrapper will help fix this, I'll explain further in the CSS.